packages/currency/src/index.js

```diff
@@ -159,124 +167,24 @@ const CurrencyFactory = ( currencySetting ) => {
 			}
 			return formatAmount( number );
 		},
+
+		/**
+		 * Format currency from WooCommerce PHP version to WCA JS version.
+		 *
+		 * @param {Object} phpCurrency Currency to format
+		 * @return {Object} Formatted JS currency.
+		 */
+		formatPhpToJs( phpCurrency ) {
```

Comment:
This feels like an odd thing to have in the public API of the currency package. Could this perhaps be a method to return the same formatted data for a given country code?

E.g.: `Currency.getDataForCountry( countryCode )` ?

Reply:
I agree it's a bit odd.  Added `Currency.getDataForCountry` but this will require passing in the locale info and symbol info since we can't guarantee access to this localized data.

Not sure if there's a better way to go about this without coupling this data to a package.